lib/airbrake/rails3_tasks.rb

```diff
@@ -59,6 +59,11 @@ def wait_for_threads
     class ActionDispatch::DebugExceptions; def call(env); @app.call(env); end; end
     class ActionDispatch::ShowExceptions; def call(env); @app.call(env); end; end
 
+    # Override BetterErrors middleware if present
```

Comment:
Because we want to show only relevant info to users.
We want to show them configuration and if task passed or not.
There is no purpose to see stack trace and other details which are displayed by other middlewares.